### DISCUSSION TITLE
Question About Sniper Working Or not

### DISCUSSION THREAD

**Discussion by AtharvaxL** [Vedza/NitroSniperGo#380] [Q&A]
is this sniper really workin or not i am getting redeemed already from 2 months and i atleast snipe 10 nitros a day.did anyone really snipe a nitro with this? please share ss if u did. cause I am kinda getting sus of this

**Comment by Zenixas** [comment]
yes it's working

**Comment by SloppyPuppy** [comment]
I used to be able to snipe, but I believe the competition has gotten fierce, and fewer codes to go around, enter less moderated servers & your ping is very important if u wanna be the first to snipe the code. Therefore someone like me on 40ms (even with a wired connection) has basically 0% chance to snipe at this point.

**Reply by AtharvaxL** [reply]
@ibndiaye how many alts do u use at most and how many servers in total

**Reply by ibndiaye** [reply]
2 alts 250+ servers
